src/main/java/seedu/club/logic/parser/ClearCommandParser.java

```diff
@@ -0,0 +1,21 @@
+package seedu.club.logic.parser;
+
+import seedu.club.logic.commands.ClearCommand;
+import seedu.club.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ClearCommand object
+ */
+public class ClearCommandParser implements Parser<ClearCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the ClearCommand
+     * and returns an ClearCommand object for execution.
+     */
+    public ClearCommand parse(String args) throws ParseException {
+        if (args.equals("")) {
+            return  new ClearCommand();
```

Comment:
Is there a reason why no arguments will be passed for empty args? This will create ClearCommand using no arguments. Hence the this.args of the ClearCommand will be null. It will throw a nullpointerexception in ClearCommand line 43.

Reply:
to be volatile without creating two command files. 
so
1. Clear
2. Clear Y
the first will take no args